2025-01-25 16:00:20,922 INFO    MainThread:27064 [wandb_init.py:setup_run_log_directory():624] Logging user logs to c:\Users\User\Documents\GitHub\mlops\wandb\run-20250125_160020-3j4phh0p\logs\debug.log
2025-01-25 16:00:20,922 INFO    MainThread:27064 [wandb_init.py:setup_run_log_directory():625] Logging internal logs to c:\Users\User\Documents\GitHub\mlops\wandb\run-20250125_160020-3j4phh0p\logs\debug-internal.log
2025-01-25 16:00:20,923 INFO    MainThread:27064 [wandb_init.py:monkeypatch_ipython():576] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000026A98C1A920>
2025-01-25 16:00:20,923 INFO    MainThread:27064 [wandb_init.py:init():743] calling init triggers
2025-01-25 16:00:20,923 INFO    MainThread:27064 [wandb_init.py:init():748] wandb.init called with sweep_config: {}
config: {}
2025-01-25 16:00:20,924 INFO    MainThread:27064 [wandb_init.py:init():765] found existing run on stack: m02352u0
2025-01-25 16:00:20,924 INFO    MainThread:27064 [wandb_run.py:_finish():2134] finishing run realx1212-aijosh/ViT_vs_CNN_jellyfish/m02352u0
2025-01-25 16:00:20,925 INFO    MainThread:27064 [wandb_run.py:_atexit_cleanup():2374] got exitcode: 0
2025-01-25 16:00:20,925 INFO    MainThread:27064 [wandb_run.py:_restore():2356] restore
2025-01-25 16:00:20,925 INFO    MainThread:27064 [wandb_run.py:_restore():2362] restore done
2025-01-25 16:00:23,313 INFO    MainThread:27064 [wandb_run.py:_footer_history_summary_info():3976] rendering history
2025-01-25 16:00:23,313 INFO    MainThread:27064 [wandb_run.py:_footer_history_summary_info():4008] rendering summary
2025-01-25 16:00:23,313 INFO    MainThread:27064 [wandb_run.py:_footer_sync_info():3937] logging synced files
2025-01-25 16:00:23,317 INFO    MainThread:27064 [wandb_init.py:init():776] starting backend
2025-01-25 16:00:23,317 INFO    MainThread:27064 [wandb_init.py:init():780] sending inform_init request
2025-01-25 16:00:23,320 INFO    MainThread:27064 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2025-01-25 16:00:23,321 INFO    MainThread:27064 [wandb_init.py:init():795] backend started and connected
2025-01-25 16:00:23,327 INFO    MainThread:27064 [wandb_run.py:_label_probe_notebook():1213] probe notebook
2025-01-25 16:00:23,328 INFO    MainThread:27064 [wandb_run.py:_label_probe_notebook():1223] Unable to probe notebook: 'cp949' codec can't decode byte 0xeb in position 90: illegal multibyte sequence
2025-01-25 16:00:23,328 INFO    MainThread:27064 [wandb_init.py:init():888] updated telemetry
2025-01-25 16:00:23,447 INFO    MainThread:27064 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-01-25 16:00:23,718 INFO    MainThread:27064 [wandb_init.py:init():967] starting run threads in backend
2025-01-25 16:00:24,023 INFO    MainThread:27064 [wandb_run.py:_console_start():2409] atexit reg
2025-01-25 16:00:24,024 INFO    MainThread:27064 [wandb_run.py:_redirect():2259] redirect: wrap_raw
2025-01-25 16:00:24,024 INFO    MainThread:27064 [wandb_run.py:_redirect():2324] Wrapping output streams.
2025-01-25 16:00:24,024 INFO    MainThread:27064 [wandb_run.py:_redirect():2349] Redirects installed.
2025-01-25 16:00:24,025 INFO    MainThread:27064 [wandb_init.py:init():1009] run started, returning control to user process
2025-01-25 16:01:14,364 INFO    MainThread:27064 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-25 16:01:14,364 INFO    MainThread:27064 [wandb_init.py:_pause_backend():541] pausing backend
2025-01-25 16:01:23,705 INFO    MainThread:27064 [wandb_init.py:_resume_backend():546] resuming backend
2025-01-25 16:01:23,714 INFO    MainThread:27064 [wandb_init.py:setup_run_log_directory():624] Logging user logs to c:\Users\User\Documents\GitHub\mlops\wandb\run-20250125_160123-hipam84o\logs\debug.log
2025-01-25 16:01:23,715 INFO    MainThread:27064 [wandb_init.py:setup_run_log_directory():625] Logging internal logs to c:\Users\User\Documents\GitHub\mlops\wandb\run-20250125_160123-hipam84o\logs\debug-internal.log
2025-01-25 16:01:23,715 INFO    MainThread:27064 [wandb_init.py:init():743] calling init triggers
2025-01-25 16:01:23,716 INFO    MainThread:27064 [wandb_init.py:init():748] wandb.init called with sweep_config: {}
config: {}
2025-01-25 16:01:23,716 INFO    MainThread:27064 [wandb_init.py:init():765] found existing run on stack: 3j4phh0p
2025-01-25 16:01:23,716 INFO    MainThread:27064 [wandb_run.py:_finish():2134] finishing run realx1212-aijosh/ViT_vs_CNN_jellyfish/3j4phh0p
2025-01-25 16:01:23,717 INFO    MainThread:27064 [jupyter.py:save_history():458] not saving jupyter history
2025-01-25 16:01:23,717 INFO    MainThread:27064 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-25 16:01:23,717 INFO    MainThread:27064 [wandb_init.py:_jupyter_teardown():558] cleaning up jupyter logic
2025-01-25 16:01:23,717 INFO    MainThread:27064 [wandb_run.py:_atexit_cleanup():2374] got exitcode: 0
2025-01-25 16:01:23,718 INFO    MainThread:27064 [wandb_run.py:_restore():2356] restore
2025-01-25 16:01:23,718 INFO    MainThread:27064 [wandb_run.py:_restore():2362] restore done
2025-01-25 16:01:25,899 INFO    MainThread:27064 [wandb_run.py:_footer_history_summary_info():3976] rendering history
2025-01-25 16:01:25,899 INFO    MainThread:27064 [wandb_run.py:_footer_history_summary_info():4008] rendering summary
2025-01-25 16:01:25,899 INFO    MainThread:27064 [wandb_run.py:_footer_sync_info():3937] logging synced files
